walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -17,7 +17,7 @@ <h2>Menu</h2>
     </div>
 
     <div id="content">
-      <h1 class="font">Laura's Portfolio</h1>
+      <h1 id="background" class="font">Laura's Portfolio</h1>
```

Comment:
You may not want to use the `font` class here considering you have it deleted in your css

Reply:
Oh that slipped, I think everything should be fixed now!